core/controllers/base_test.py

```diff
@@ -556,24 +556,16 @@ def test_keys_in_source_code_match_en(self):
             os.path.join('core', 'templates', 'dev', 'head'),
             'extensions']
         files_checked = 0
-        missing_keys_count = 0
         for directory in dirs_to_search:
             for root, _, files in os.walk(os.path.join(os.getcwd(), directory)):
                 for filename in files:
                     if filename.endswith('.html'):
                         files_checked += 1
                         html_key_list = self._extract_keys_from_html_file(
                             os.path.join(root, filename))
-                        if not set(html_key_list) <= set(en_key_list): #pylint: disable=unneeded-not
-                            self.log_line('ERROR: Undefined keys in %s:'
-                                          % os.path.join(root, filename))
-                            missing_keys = list(
-                                set(html_key_list) - set(en_key_list))
-                            missing_keys_count += len(missing_keys)
-                            for key in missing_keys:
-                                self.log_line(' - %s' % key)
-                            self.log_line('')
-        self.assertEqual(missing_keys_count, 0)
+                        self.assertLessEqual(
+                            set(html_key_list), set(en_key_list))
```

Comment:
Will this include logs of the missing keys? That's important for the developer debugging the issue. (In general, don't reduce functionality in the existing code -- that's a regression.)

Reply:
Done